piq/functional/colour_conversion.py

```diff
@@ -113,3 +113,23 @@ def rgb2yiq(x: torch.Tensor) -> torch.Tensor:
         [0.2115, -0.5227, 0.3112]]).t().to(x)
     x_yiq = torch.matmul(x.permute(0, 2, 3, 1), yiq_weights).permute(0, 3, 1, 2)
     return x_yiq
+
+
+def rgb2lhm(x: torch.Tensor) -> torch.Tensor:
+    r"""Convert a batch of RGB images to a batch of LHM images
+
+    Args:
+        x: Batch of 4D (N x 3 x H x W) images in RGB colour space.
+
+    Returns:
+        Batch of 4D (N x 3 x H x W) images in LHM colour space.
```

Comment:
Previously you used `B` to denote batch size, but here `N` is used. It would be good to use the same letter for consistency

Reply:
However, all functions at `colour conversion` use `N`. I suggest to leave this to separate PR to unify the notation among all files of the library.